msg/ZFS-8000-ER/index.html

```diff
@@ -135,13 +135,99 @@
 
 <pre>
 # zpool get freeing
-NAME   PROPERTY  VALUE    SOURCE
-rpool  freeing   0        default
+NAME  PROPERTY  VALUE    SOURCE
+test  freeing   0        default
 </pre>
 
 The ZFS packages may be now be updated and the pool imported.  The on-disk
 format incompatibility can now be corrected online as described in
 <a href="#1">Errata #1</a>.
 </pre>
 </td></tr></table>
+<br><br>
+<hr>
+<br><br>
+<!-- end: templates/success -->
+<!-- start: article -->
+<!-- code: ZFS-8000-ER -->
+<!-- keys: errata -->
+<table width="80%"><tr><td>
+<a name="2"><b>ZFS Errata #3</b></a>
+<p>
+<dl>
+<dt><p><b>Type</b>
+<dd>Compatibility
+<dt><p><b>Severity</b>
+<dd>Moderate
+<dt><p><b>Description</b>
+<dd>The ZFS packages were updated after creating an encrypted dataset which
```

Comment:
Why specify "The ZFS packages were updated after..." here?  

I think I understand what you mean - the buggy version of ZFS wouldn't emit this message.  I think it would be clearer to say "An encrypted dataset contains an on-disk format incompatability", but maybe I'm missing something.

Reply:
I chose this wording to match the other errata. I can change it.